service/api.go

```diff
@@ -560,9 +561,36 @@ func (as *APIServer) validateProjectConfig(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
-	errs := validator.CheckYamlStrict(input.ProjectYaml)
+	errs := validator.ValidationErrors{}
+	if input.ProjectID != "" {
+		projectRef, err := model.FindMergedProjectRef(input.ProjectID, "", false)
```

Comment:
Can you actually move looking up the projectId from the local config to here? If we end up needing to lookup the project from the local config, we shouldn't error on not finding it; we should instead use the "no project specified" error. cc @hadjri feel free to LGTM this change while I'm gone

Reply:
I think the lookup has to be done in the operations package and can't be done in the API layer - I also don't see that we are erroring on not finding it. The project id will remain empty and we will get to the "no project specified" error.